sched/wqueue/wqueue.h

```diff
@@ -118,6 +128,27 @@ extern struct lp_wqueue_s g_lpwork;
  * Public Function Prototypes
  ****************************************************************************/
 
+static inline_function FAR struct kwork_wqueue_s *work_qid2wq(int qid)
+{
+#ifdef CONFIG_SCHED_HPWORK
+  if (qid == HPWORK)
```

Comment:
we should support fallback policy to keep the behavior as currently,  eg. if qid == HPWORK but only LPWORK is enabled, we should return LPWORK